src/main/java/com/hbm/entity/missile/EntityMissileCustom.java

```diff
@@ -293,7 +296,59 @@ public void onUpdate() {
 			for(int i = 0; i < velocity; i++)
 				MainRegistry.proxy.spawnParticle(posX - v.xCoord * i, posY - v.yCoord * i, posZ - v.zCoord * i, smoke, null);
 		}
-
+		
+		
+		if(type1 == WarheadType.MIRV){
+			
+    		if((motionY <= 0) && (posY <= 300)) {
+    			
+    			if(worldObj.isRemote)
+					return;    
+    			               
+    				this.setDead();
+    				           
+    				double mod;
+    				double mod2;
+    				for(int i = 0; i < 7; i++) {
+    					EntityMIRV nuke3 = new EntityMIRV(this.worldObj);
+    					nuke3.setPosition(posX,posY,posZ);      
+    					mod = (i==(1 & 2)) ? 1 : -1; 
```

Comment:
`&` is a bit operator. did you mean `i == 1 || i == 2`?

Reply:
Yup, that explains why I had a massive headache trying to make the +X +Z quadrant mirvlet work